src/WorkspaceUpdater.ts

```diff
@@ -28,11 +29,30 @@ export class WorkspaceUpdater implements vscode.Disposable {
   }
 }
 
+async function sendEvent(event: string, payload: Record<string, unknown> = {}) {
+  if (isCapabilityEnabled(Capability.WORKSPACE_TRACE)) {
+    fireEvent({
+      name: event,
+      ...payload,
+    });
+  }
+}
```

Comment:
you can add the `void` here and get rid of it in the callers
```suggestion
function sendEvent(event: string, payload: Record<string, unknown> = {}) {
  if (isCapabilityEnabled(Capability.WORKSPACE_TRACE)) {
    void fireEvent({
      name: event,
      ...payload,
    });
  }
}
```